Data Structures/Linked Lists/C++/Doubly_Linked_List.cpp

```diff
@@ -0,0 +1,91 @@
+#include <iostream>
+
+using namespace std;
+int main()
+{
+struct node{
+	int info;
+	node *left,*right;
+}*ptr,*start,*last,*save;
+
+int c=1,i=0,data,item;
+start=last=NULL;
+
+while(c<4 && c>0){
+	cout<<"1.Insert\n2.Deletion\n3.Linked List\n";
```

Comment:
Add insert and delete functions. Do not put all your code in a single switch statement